src/Kaiko-Api-Client/openApi3.yaml

```diff
@@ -0,0 +1,260 @@
+﻿openapi: 3.0.0
+info:
+  title: Kaiko API
+  version: '1.0.0'
+servers:
+  - url: 'https://eu.market-api.kaiko.io/'
+    description: Kaiko base url
+  - url: 'https://reference-data-api.kaiko.io/v1/'
+    description: Override base path for all operations with the /Instruments path
+paths:
+  https://reference-data-api.kaiko.io/v1/instruments:
```

Comment:
can you make that path relative?

Reply:
At the begginning, it was relative, but I didn't find a way to select multiple base path. 
Instruments and exchanges have the same base path but it's not the same as the one for getVwap.

And Instruments and exchanges are useful to get the supported exchanges and currencies to put in parameter of getVwap...

Reply:
OK let me have a look :) we might need to have 2 configurators and select the one with correct base path on injection, or have 2 swagger files...

Reply:
I don't know what is best, to have simply 2 swagger files or to go through the pain of adding url on each path...
probably 2 swagger files is OK

Reply:
Yes, I've seen these links yesterday but impossible to generates different bath path. I look into all issues in nswag by Rico but didn't find anything else...

I think that I'm gonna make two swagger files...